### PR TITLE
CLI: implement 'openio-admin container move' command

### PR DESCRIPTION
**Pull request by AymericDu**
##### SUMMARY

Implement 'openio-admin container move' command.

##### ISSUE TYPE

 - Feature Pull Request
 - Bugfix Pull Request

##### COMPONENT NAME

- `Python API`
- Admin CLI

##### SDS VERSION

```
openio 4.4.0.0b2.dev19
```

### CODE REVIEW

File: oio/directory/meta2.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
+# Copyright (C) 2018-2019 OpenIO SAS, as part of OpenIO SDS
```
Comment (kamel-rahim) [inline oio/directory/meta2.py]: Why 2018?

Comment (AymericDu) [reply oio/directory/meta2.py]: Part of the code in this file is already used in `oio/directory/meta1.py`.
And `oio/directory/meta2.py` replace `oio/directory/meta1.py`.

File: oio/directory/meta2.py [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+                base, src, dst, exc)
+            yield {'base': base, 'src': src, 'dst': dst, 'err': exc}
+            return
+
+        for base, err in bases_with_error:
+            if err is None:
+                base, src, dst, err = self._safe_move(base, src, dst)
+                yield {'base': base, 'src': src, 'dst': dst, 'err': err}
+            else:
+                yield {'base': base, 'src': src, 'dst': dst, 'err': err}
```
Comment (murlock) [inline oio/directory/meta2.py]: `dst` may be still `None`, is it ok ?

Comment (AymericDu) [reply oio/directory/meta2.py]: It's ok.
However, you put your finger on a small duplication of the code.

Comment (murlock) [reply oio/directory/meta2.py]: In fact, `dst` should be reset to avoid reusing previous `dst` ?

**Comment by fvennetier** [comment]
I did some changes, but some parts are still unclear.

**Pull request merged**